config/suppressions-xpath.xml

```diff
@@ -515,4 +515,8 @@
           checks="LocalVariableNameCheck"
           query="//METHOD_DEF[./IDENT[@text='getLineNumbersFromExpected']]
                 //VARIABLE_DEF/IDENT[@text='i']"/>
+  <suppress-xpath
+          files="AbstractRootNodeTest.java"
+          checks="AbstractClassName"
```

Comment:
not good, see a reason in another comment.

Reply:
Done